README.md

```diff
@@ -627,6 +630,37 @@ ServiceNow.getAllAttachmentsMetaData('a83820b58f723300e7e16c7827bdeed2', (res) =
 ___
 
 
+### 12. ServiceNow.getChangeTasks
+
+This is used to get change tasks from a change request in ServiceNow. (#12-getchangetasks)
+
+| Parameters        | Description                                                |
+|-------------------|------------------------------------------------------------|
+| type              | type of table - change_request                             |
+| changeNumber      | Change request number                                      |
+| callback function | Response will be available as a parameter to this function |
+
+#### Request
+
+```
+ServiceNow.createNewTask('change_task', 'CHG0000016', res=>{
+    console.log(res);
+});
```

Comment:
This is wrong example for fetch tasks.. can replace it with below

```
ServiceNow.getChangeTasks('change_task', 'CHG0000001', (res) =>
    console.log(res)
);
```

Reply:
wrong copy past :) fixed